### PR TITLE
Make this module consumable for projects not using webpack

### PR DESCRIPTION
**Pull request by ivantm**
If I attempt to import this module into my project (which only uses rollup) via the following code:

``` js
import SVG from 'preact-svg';
```

then rollup reports the following error:

```
preact.js does not export Component 
```

Although Webpack permits destructuring in imports from ES5 modules in various formats, Rollup does not. This module is expecting Webpack-like functionality in `src/index.js`:

``` js
import { h, Component } from 'preact';
```

Even though this project uses rollup to build its distributable, it excludes preact in `rollup.config.js`. I can cause the error described above by removing the `external` config in the main config object and `skip: external` in the `nodeResolve` config nested further in.... which means the reason the import works is because rollup is ignoring it. The tests pass because the Karma config includes a webpack config, which then makes the import in the test file work.

tl;dr in order to make this project consumable without webpack, we need to change

``` js
import { h, Component } from 'preact';
```

into 

``` js
import preact from 'preact';
const { h, Component } = preact;
```

### CODE REVIEW

**Comment by developit** [comment]
@ivantm doesn't setting `rollup-plugin-node-resolve` to use `jsnext:main` fix this? Rollup should handle both modules by using their es6 source.

What version of preact did you try this with by the way? The ES Modules export has been fixed in beta.

**Comment by ivantm** [comment]
> What version of preact did you try this with by the way?

I'm using the current stable release (4). I see 3 options that could work:
- do a new release of preact 4 which includes the additional `export` code that has been added in beta 5. This would make this PR unnecessary.
- merge this, and then revert for a future version of `preact-svg` which is compatible with preact 5.
- I upgrade to the beta version with fingers crossed 😄 

`preact-svg` has `preact: *` in package.json - this could be dangerous if/when future breaking changes are released.

**Comment by developit** [comment]
@ivantm the interesting thing here is that `preact-svg` is actually unnecessary as of preact 5, since 5 ships with native SVG support built in.  If you're looking to update to the beta, feel free to ping me on [Gitter](https://gitter.im/developit/preact), I can help if you run into any issues.  It should be stable at this point, I just am being risk-averse ;)

**Comment by ivantm** [comment]
I'll take 5 for a spin. Thanks!

**Comment by developit** [comment]
@ivantm Great! There is one bug that may or may not affect you, but I have a fix locally and will be publishing it within the hour. Just doing some performance testing prior to publish.  I'm actually considering publishing 5.1.0 as stable this evening, so don't worry too much about using it under the beta flag :)

**Comment by developit** [comment]
@ivantm 5.3.0 is now released and out of beta, making `preact-svg` obsolete, so I think we can close this. Cheers!

**Comment by ivantm** [comment]
Thanks! Love your work 💯
